geonode/upload/forms.py

```diff
@@ -86,11 +95,36 @@ def clean(self):
         cleaned["valid_extensions"] = valid_extensions
         return cleaned
 
+    def validate_files_sum_of_sizes(self):
+        max_size = self._get_uploads_max_size()
+        total_size = self._get_uploaded_files_total_size()
+        if total_size > max_size:
+            raise forms.ValidationError(_(
+                f'Total upload size exceeds {filesizeformat(max_size)}. Please try again with smaller files.'
+            ))
+
+    def _get_uploads_max_size(self):
+        try:
+            max_size_db_obj = UploadSizeLimit.objects.get(slug="total_upload_size_sum")
+        except UploadSizeLimit.DoesNotExist:
+            max_size_db_obj = UploadSizeLimit.objects.create_default_limit()
+        return max_size_db_obj.max_size
+
     def _get_uploaded_files(self):
         """Return a list with all of the uploaded files"""
         return [django_file for field_name, django_file in self.files.items()
                 if field_name != "base_file"]
 
+    def _get_uploaded_files_total_size(self):
+        """Return a list with all of the uploaded files"""
+        excluded_files = ("zip_file", )
+        uploaded_files_sizes = [
```

Comment:
@italogsfernandes the issue looks resolved for the zip files, but is still present for the scenario where u want to upload the files without a zip (providing shp, shx, etc).
The base_file key is present in the same file (usually) as the shp key, so in this case, the total sum of the files is not correct.
Those are the available files in the `self.files.items()`, as u can see the shp file is duplicated:
```python
0:<SimpleUploadedFile: ProvCM01012019_g_WGS84.shp (application/x-qgis)>
1:<SimpleUploadedFile: ProvCM01012019_g_WGS84.dbf (application/x-dbf)>
2:<SimpleUploadedFile: ProvCM01012019_g_WGS84.prj (application/octet-stream)>
3:<SimpleUploadedFile: ProvCM01012019_g_WGS84.shp (application/x-qgis)>
4:<SimpleUploadedFile: ProvCM01012019_g_WGS84.shx (application/x-qgis)>
```
Can we add few test on this part?